app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/data/CoInsured.kt

```diff
@@ -1,11 +1,22 @@
 package com.hedvig.android.feature.editcoinsured.data
 
+import java.time.format.DateTimeFormatter
 import kotlinx.datetime.LocalDate
+import kotlinx.datetime.toJavaLocalDate
 
 data class CoInsured(
   val firstName: String?,
   val lastName: String?,
   val birthDate: LocalDate?,
   val ssn: String?,
   val hasMissingInfo: Boolean,
-)
+) {
+  val displayName: String? = if (firstName != null && lastName != null) {
+    "$firstName $lastName"
+  } else {
+    null
+  }
+
+  fun details(dateTimeFormatter: DateTimeFormatter): String? =
```

Comment:
Not sure how `details` is used here, maybe rename this function to a more descriptive name? Otherwise, if it's only used in 1 very specific place where it makes sense only there, I wouldn't mind it to just be inlined there instead.